packaging/standalone/standalone-community/src/main/distribution/text/community/UPGRADE.txt

```diff
@@ -1 +1 @@
-For upgrade instructions, please see http://neo4j.com/docs/operations-manual/current/#upgrading.
+For upgrade instructions, please see http://neo4j.com/docs/operations-manual/current/.
```

Comment:
I think this is the new link: http://neo4j.com/docs/operations-manual/current/deployment/#upgrade